spec/features/incoming_mail_handler_spec.rb

```diff
@@ -49,21 +50,25 @@
     mandrill_event = mandrill_payload_from_message(mail)
     mandrill_event[0]['msg']['from_email'] = user.email_address
     handler = IncomingMail::Handler.new
-    expect(my_approval.proposal.existing_observation_for(user)).to be_falsey
-    expect(my_approval.proposal.existing_approval_for(user)).to be_falsey
+
+    expect(my_approval.proposal.existing_observation_for(user)).not_to be_present
+    expect(my_approval.proposal.existing_approval_for(user)).not_to be_present
+
     resp = handler.handle(mandrill_event)
+
     expect(resp.action).to eq(IncomingMail::Response::COMMENT)
-    expect(resp.comment.proposal.existing_observation_for(user)).to be_truthy
-    expect(resp.comment.proposal.existing_approval_for(user)).to be_falsey
+    expect(resp.comment.proposal.existing_observation_for(user)).to be_present
+    expect(resp.comment.proposal.existing_approval_for(user)).not_to be_present
     expect(my_approval.user).to_not eq(my_proposal.individual_approvals.last.user)
-    recipients = [
-      user.email_address, # observer notification
```

Comment:
this was removed intentionally - we are sending a comment email to all of the other people watching the proposal. We are no longer sending an "observation notification" email because the way the logic is set up, an observer added email is only sent when there is an "adder" . But, this logic can be changed. Thoughts?